src/app/community-list-page/community-list/community-list.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, OnInit } from '@angular/core';
+import { CommunityListService, FlatNode } from '../community-list-service';
+import { CommunityListDatasource } from '../community-list-datasource';
+import { FlatTreeControl } from '@angular/cdk/tree';
+import { isEmpty } from '../../shared/empty.util';
+
+@Component({
```

Comment:
TypeDocs please at the Class level (most of the methods already have comments, and that's great to see)